app/views/admin/participants/_cohorts.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= govuk_warning_text do %>
+  This is a temporary tab intended to help debug the multiple cohort problems.
+<% end %>
+
+<%= govuk_summary_list do |sl|
+  sl.row do |row|
+    row.key(text: "Cohort (via induction record)")
+    row.value(text: latest_induction_record.cohort.start_year)
+  end
```

Comment:
From `InductionRecord`:
```
has_one :school_cohort, through: :induction_programme
has_one :cohort, through: :school_cohort
```
so Cohort (via induction_record) and Cohort (via induction_programme) are the same